### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by jzbz**
We were linking to Core's README.md

### CODE REVIEW

**Comment by sandakersmann** [comment]
ACK

File: doc/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The following are developer notes on how to build Bitcoin on your native platfor
 
 Development
 ---------------------
-The Bitcoin repo's [root README](https://github.com/bitcoin/bitcoin/blob/master/README.md) contains relevant information on the development process and automated testing.
+The Bitcoin repo's [root README](https://github.com/bitcoinclassic/bitcoinclassic/blob/master/README.md) contains relevant information on the development process and automated testing.
```
Comment (maflcko) [inline doc/README.md]: What about `[root README](/README.md)`?

**Comment by jzbz** [comment]
Fixed, thanks @MarcoFalke.

File: doc/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The following are developer notes on how to build Bitcoin on your native platfor
 
 Development
 ---------------------
-The Bitcoin repo's [root README](https://github.com/bitcoin/bitcoin/blob/master/README.md) contains relevant information on the development process and automated testing.
+The Bitcoin repo's [root README](/README.md) contains relevant information on the development process and automated testing.
```
Comment (maflcko) [inline doc/README.md:50]: @jzbz  Mind to squash the two commits?

**Pull request closed without merging**